### ISSUE TITLE
Resuming Music After Playing a Clip?

### ISSUE THREAD

**Issue by AlexKLMan** [jishi/node-sonos-http-api#901]
Hi there, loving this program so far, I have built a fake occupancy sound with this system.
What I'm having issue with is resuming my music when a clip plays, I tried using /resume, but it just plays the clip from my clips directory again and not the song that was playing.
Am I using the wrong commands?
Thank you.

**Comment by jsiegenthaler** [comment]
I think that is a Sonos limitation. If it was air-playing or streaming Sonos tracks, and you switch to a local mp3 clip, then resume does nothing. If you are playing a mp3 clip, then make an announcement using TTS then resume will work and the mp3 clip will play again.

**Comment by stevegroom** [comment]
I’ve been using the TTS (Text to Speech) service with my SONOS speakers and that interrupts my playing music, speaks the alert and then resumes the music.

**Comment by AlexKLMan** [comment]
@jsiegenthaler Yeah I'm playing on Spotify, not a local mp3, what's the best way to play the local mp3?

**Comment by AlexKLMan** [comment]
@stevegroom Hmm it looks like the same issue I use http://192.168.50.50:5005/Sonos%20Play%205/say/Dinner%20is%20ready/en-gb/50
It will speak in the Google voice, but then does not resume my music on Spotify.

**Comment by protechpdx** [comment]
clip takes over the queue, so resume doesn't work.
say seems to skip to the next track, at least with Apple Music, which is better but still not ideal.

Did anybody ever find a solution? I know the official Sonos cloud API has the option to tag something as an announcement and duck the currently playing media without actually stopping it, but not sure if that's possible here.
